### PR TITLE
Portfolio adjustments

### PR DESCRIPTION
**Pull request by GottaHackemAll**

### CODE REVIEW

File: MapStyle.css [hunk 1]
```diff
@@ -2,61 +2,270 @@
...
+        left: 77%;
+        top: 32.4%;
+
+    }
+
+    #pin9 {
+        height: 25px;
+        left: 52.5%;
+        top: 57%;
+
```
[the remark this answers is not in the document]
Comment (GottaHackemAll) [reply MapStyle.css]: I'm having as much fun deleting these as you had creating them.

File: About.html [hunk 2]
```diff
@@ -2,22 +2,30 @@
...
+                <a href="map.html">
+                    <img class="globe" src="images/globe.png">
                 </a>
-                <a class="nav-item" href="http://localhost:63342/portfolio/projects.html?_ijt=ptrkkbs1r9r67l2rbnf4brv3a1">Projects</a>
-                <a class="nav-item" id="contact" href="#">Contact Me</a>
+                <a class="nav-item"
+                   href="project.html">Projects</a>
+                <div class="contact-window">
+                    <a href="#">
+                        <img class="contact-icon" src="images/linkedin.png">
```
Comment (sjr-mayden) [inline About.html]: Please give alts for those who can't see the picture clearly

File: MapStyle.css [hunk 3]
```diff
@@ -2,61 +2,304 @@
...
 }
 
+body {
+    margin: 0;
+    padding: 0;
+    font-family: Helvetica;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
```
Comment (sjr-mayden) [inline MapStyle.css]: Extra newline here

File: MapStyle.css [hunk 4]
```diff
@@ -2,61 +2,304 @@
...
+    left: 45.9%;
+    top: 17.5px;
+    width: 80px;
+    height: 80px;
+    border: solid #E53761;
+    border-radius: 100%;
+    transition: transform .5s ease-in-out;
+}
+
+
```
Comment (sjr-mayden) [inline MapStyle.css]: Extra newline here

File: MapStyle.css [hunk 5]
```diff
@@ -2,61 +2,304 @@
...
+.globe:hover {
+    transform: rotate(360deg);
+}
+
+
```
Comment (sjr-mayden) [inline MapStyle.css]: Extra newline here

File: MapStyle.css [hunk 6]
```diff
@@ -2,61 +2,304 @@
...
+    height: 25px;
+    left: 52%;
+    top: 74%;
 }
+
+@media only screen and (max-width: 1000px) {
+    .container {
+        width: 800px;
+    }
+    .nav-item {
```
Comment (sjr-mayden) [inline MapStyle.css]: Missing newlines here

File: project.html [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+                        dolore
+                </div>
+            </div>
+            <div class="border-divider">
+                <div class="project-right">
+                    <a href="#">
+                        <img class="project-img-right" src="https://hungarytoday.hu/wp-content/uploads/2018/02/18ps27.jpg">
+                    </a>
+                    <h3><u>Project 4</u></h3>
+                    <p class="preject-text-right">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
```
Comment (sjr-mayden) [inline project.html]: This looks like PHPStorm's auto formatting has misunderstood what you're trying to do here, probably best to just make it neat yourself

**Pull request merged**